cmd/diagnostic/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM docker:17.12-rc-dind
+
+RUN apk add --no-cache curl
+
+WORKDIR /tool
+
+COPY daemon.json /etc/docker/daemon.json
+COPY diagnosticClient /tool/diagnosticClient
```

Comment:
For a follow-up; we should make this Dockerfile a multi-stage build, and actually build the client in it (I had a branch with that, but looks like I didn't push, and it's not on my laptop 😊